packages/components/src/Persona.purs

```diff
@@ -132,11 +132,13 @@ temporaryAddressChange
   -> Aff Subscription
 temporaryAddressChange uuid subsno startDate endDate streetAddress zipCode countryCode temporaryName token = do
   let startDateISO = formatDate startDate
-      endDateISO   = formatDate endDate
+      endDateISO   = case endDate of
```

Comment:
`Maybe a` is an instance of the `Functor` typeclass, that is, it can be mapped over, meaning that if you want to get a `Maybe b` value from a `Maybe a` value by applying a function, you can use `fmap`, aka `<$>`.

I suggest you change this to
```
formatDate <$> endDate
```

Reply:
4d7c964